k8s/src/main/scala/io/buoyant/k8s/Watchable.scala

```diff
@@ -83,19 +83,19 @@ private[k8s] abstract class Watchable[O <: KubeObject: TypeReference, W <: Watch
    *
    * TODO: investigate k8s websockets support as an alternative HTTP approach
    *
-   * @return An asynchronous stream of [[Watch]] objects containing additions, modifications, and
-   *         deletions to the items in the list being watched, and a Closable handle allowing for
-   *         the watch to be terminated.
+   * @return An update function that may be passed to [[Var.async]] to create stream of [[Watch]] objects containing
+   *         additions, modifications, and deletions to the items in the list being watched, and a Closable handle
+   *         allowing for the watch to be terminated.
```

Comment:
nit: i suspect these comment lines are a big longer than the usual 100 char limit we like to apply.

Would you mind wrapping comments at 100 chars here & throughout?  Thanks!